arb_os/evmCallStack.mini

```diff
@@ -1102,6 +1108,7 @@ public impure func evmCallStack_callRetryableTxAndReturn(rtx: RetryableTx) -> (b
             bytearray_size(rtx.calldata),
             0,
             0,
+            false,
```

Comment:
Confirming that the behavior of retryables sent to the 0 address is not to call it as a constructor. We should clarify the documentation if it isn't already clear

Reply:
Yes, a retryable sent to address zero will be treated as a tx to address zero, not as a deploy.  There shouldn't be any way to create a retryable that is a deploy.

Reply:
We should clarify that there's no way to use a retryable to deploy a contract since I think that's likely non-intuitive